packages/explat-client-react-helpers/src/test/index.tsx

```diff
@@ -62,13 +62,64 @@ describe( 'useExperiment', () => {
 		);
 
 		expect( result.current ).toEqual( [ true, null ] );
+		expect(
+			( exPlatClient.loadExperimentAssignment as jest.MockedFunction<
+				typeof exPlatClient.loadExperimentAssignment
+			> ).mock.calls.length
+		).toBe( 1 );
```

Comment:
Could you use a dedicated Jest assert here instead?
```js
expect( explatClient.loadExperimentAssignment ).toHaveBeenCalledTimes( 1 );
```

Reply:
Oh nice! I didn't know that existed 😄